tests/unit/s2n_psk_key_exchange_modes_extension_test.c

```diff
@@ -245,5 +245,50 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_stuffer_free(&out));
     };
 
+    /* Test should send */
+    {
```

Comment:
Nit: let's match the style of the rest of this test suite. Move this above the self-talk test, and name it "Test: s2n_psk_key_exchange_modes_should_send". Even more nitpicky, the other tests use Sentence case for the names of their test cases, and you're using all lowercase.